adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -24,25 +24,26 @@
  */
 @property(nonatomic, copy) NSString *_Nullable userId;
 
+@property(nonatomic, copy) NSString *_Nullable playingPlacement;
+
 /*!
- * @brief Controls whether presented ads will start in a muted state or not.
+ * @brief NSInteger that will be passed to alter the privacy icon position for native ads.
+ * @discussion Optional. topLeft = 1, topRight = 2, bottomLeft = 3, bottomRight = 4
  */
-@property(nonatomic, assign) BOOL muted;
+@property(nonatomic, assign) NSInteger nativeAdOptionPosition;
 
-@property(nonatomic, assign) NSUInteger ordinal;
+@property(nonatomic, assign) BOOL muted DEPRECATED_ATTRIBUTE;
 
-@property(nonatomic, assign) NSTimeInterval flexViewAutoDismissSeconds;
+@property(nonatomic, assign) NSUInteger ordinal DEPRECATED_ATTRIBUTE;
 
-@property(nonatomic, copy) NSArray<NSString *> *_Nullable allPlacements;
-
-@property(nonatomic, copy) NSString *_Nullable playingPlacement;
+@property(nonatomic, assign) NSTimeInterval flexViewAutoDismissSeconds DEPRECATED_ATTRIBUTE;
 
-@property(nonatomic, copy) NSNumber *_Nullable orientations;
+@property(nonatomic, copy) NSArray<NSString *> *_Nullable allPlacements DEPRECATED_ATTRIBUTE;
 
-@property(nonatomic, copy, readonly) NSString *_Nonnull UUID;
+@property(nonatomic, copy) NSNumber *_Nullable orientations DEPRECATED_ATTRIBUTE;
 
-@property (nonatomic, readonly, assign) BOOL muteIsSet;
+@property(nonatomic, copy, readonly) NSString *_Nonnull UUID DEPRECATED_ATTRIBUTE;
 
-@property (nonatomic, assign) NSInteger nativeAdOptionPosition;
+@property (nonatomic, readonly, assign) BOOL muteIsSet DEPRECATED_ATTRIBUTE;
```

Comment:
Since all of these properties have been marked deprecated, can you confirm if they aren't supported anymore? Are there substitutions?

(If they aren't supported anymore, this section in the dev doc won't be valid anymore: https://developers.google.com/admob/ios/mediation/vungle#network-specific_parameters)

Reply:
Yes. Those properties are no longer supported anymore without any substitutions.